addons/mail/models/mail_activity.py

```diff
@@ -60,24 +60,26 @@ def default_get(self, fields):
         domain=['&', ('is_mail_thread', '=', True), ('transient', '=', False)],
         help='Specify a model if the activity should be specific to a model'
              ' and not available when managing activities for other models.')
-    default_next_type_id = fields.Many2one('mail.activity.type', 'Default Next Activity',
+    default_next_type_id = fields.Many2one('mail.activity.type', string='Force Next',
```

Comment:
Since the schema is being migrated and you're renaming fields anyway, shouldn't this field be renamed? "Force next" and "Recommend next" as field names are also less than clear.

Reply:
Ok, I'll rename the fields, "Force next" and "Recommend next" where the placeholder requested from (LNA), do you think "Force next type" and "Recommend next type" would be more appropriate ? I will also discuss this with (LNA)